### DISCUSSION TITLE
[求助]新手绳匠如何快速提高绳网等级？

### DISCUSSION THREAD

**Discussion by delete-slash** [share121/inter-knot#217] [其他]
如题~刚当上绳匠没多久，在绳网上接不到单，绳网等级提高不了
<img width="70" alt="屏幕截图 2024-02-16 224634" src="https://github.com/user-attachments/assets/8b4cce80-0494-494a-998e-123e0c43f709">

**Comment by 102266270** [comment]
爆女装图可以直接提升到50级，真的有效，不信来空洞砍我

**Comment by WORLD-executeME** [comment]
多接委托多水贴

**Comment by Cocoon13022195** [comment]
当然是去找启明星大姐姐接委托啦！

**Comment by SDCOM-0415** [comment]
我上线就26级了

**Comment by AL-Pinecore** [comment]
答: 开50个Repository，直升50级
